nengo/tests/test_processes.py

```diff
@@ -99,7 +100,7 @@ def test_gaussian_whitenoise(Simulator, rms, seed, plt):
 def test_whitesignal_rms(Simulator, rms, seed, plt):
     t = 1.
     d = 500
-    process = WhiteSignal(t, rms=rms)
+    process = WhiteSignal(t, high=500, rms=rms)
```

Comment:
Yeah, I guess it's just the way I've always done it. I'm used to programming languages where all you have is parameter order. But you're right, it does help with readability, so it's probably good to keep it the way it is. But why don't we do the same for the period, then?